rust/rsc/src/database.rs

```diff
@@ -249,7 +249,7 @@ pub async fn oldest_jobs<T: ConnectionTrait>(db: &T) -> Result<Vec<OldestJobs>,
         INNER JOIN job j
         ON j.hash = h.hash
         ORDER BY j.created_at
-        LIMIT 50
+        LIMIT 30
```

Comment:
This change makes me think it would be useful to be able to get more or less of these as part of the request parameters, but that seems like it would add a lot of complications, and is relatively low priority

Reply:
Yeah, I'd like to parameterize it eventually but I think for a first pass I'm happy with this being constant